src/index.html

```diff
@@ -11,8 +11,62 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The stylesheet link references a '.scss' file. Ensure that this SASS file is compiled to CSS, and link the resulting '.css' file instead, as browsers cannot directly interpret SASS files.